docs/projects/amt/reporting-standard/latest.md

```diff
@@ -151,7 +151,7 @@ There can be multiple instruments. For each instrument the following fields are
 11. `labels` (OPTIONAL, list). This field allows to store meta information about a system. There can be multiple labels.
    For each label the following fields are present.
 
-    1. `name` (OPTIONAL, string). Name of the label.
+    1. `name` (REQUIRED, string). Name of the label.
     2. `value` (OPTIONAL, string). Value of the label.
```

Comment:
```suggestion
    2. `value` (REQUIRED, string). Value of the label.
```
It doesn't make so much sense if you only have the name of a label but not its actual value, so also make this REQUIRED if you have the name?

Reply:
I thought we discussed this morning that only the name is mandatory, as it is not the same as a traditional key-value pair. Also this comes back in the designs that have been made.

Reply:
True the others are not traditional key-value pairs but this one actually is i have a feeling